src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Properties/InterceptedProjectProperties/ApplicationPropertyPage/CompoundTargetFrameworkValueProvider.cs

```diff
@@ -28,14 +30,16 @@ internal sealed class CompoundTargetFrameworkValueProvider : InterceptingPropert
         private readonly ConfiguredProject _configuredProject;
         private bool? _useWPFProperty;
         private bool? _useWindowsFormsProperty;
-
+        private bool? _useWinUIProperty;
         private static readonly string[] s_msBuildPropertyNames = { TargetFrameworkProperty, TargetPlatformProperty, TargetPlatformVersionProperty, SupportedOSPlatformVersionProperty };
+        private static readonly Regex s_versionRegex = new(@"^net(?<version>[0-9.]+)$", RegexOptions.ExplicitCapture);
```

Comment:
Is there any chance the value would have a suffix, such as `net7.0-windows`?

Reply:
Not really, since we are working here with targetFrameworkAlias, that's only the first part. Since we get them from https://github.com/dotnet/sdk/blob/main/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.SupportedTargetFrameworks.props, I think we are good.